doc/DOCUMENTATION.md

```diff
@@ -9,36 +9,34 @@ project.
 Table of contents
 -----------------
 
- - [What is Lambdapi?](sections/about.md)
+ - [What is Lambdapi?](about.md)
 
- - [Installation](sections/install.md)
+ - [Installation](install.md)
 
- - [Getting started](sections/getting_started.md)
+ - [Getting started](getting_started.md)
 
- - [Command line options](sections/options.md)
+ - [Command line options](options.md)
 
- - [User interfaces](sections/ui.md)
+ - [User interfaces](ui.md)
 
- - [Module system](sections/module.md)
+ - [Module system](module.md)
 
- - [Syntax of terms](sections/terms.md)
+ - [Syntax of terms](terms.md)
 
- - [Commands](sections/commands.md)
+ - [Commands](commands.md)
 
- - [Tactics](sections/tactics.md)
+ - [Tactics](tactics.md)
 
- - [Editing Lambdapi source code with Emacs](sections/emacs.md)
```

Comment:
There is no more entry for emacs, is it wanted?

Reply:
Yes, it is moved into ui.md.

Reply:
So personally, I'd prefer to keep a slightly longer table of content (it's not that long to read...) to have more accessible pages and less "click paths" to reach a page.

Reply:
Good point. I gave more structure to the toc. Is it better now?

Reply:
Yes, that's nice